core/templates/dev/head/pages/admin/Admin.js

```diff
@@ -17,11 +17,13 @@
  */
 
 oppia.constant('ADMIN_HANDLER_URL', '/adminhandler');
+oppia.constant('ADMIN_ROLE_HANDLER_URL', '/adminrolehandler');
 oppia.constant('PROFILE_URL_TEMPLATE', '/profile/<username>');
 oppia.constant(
   'ADMIN_JOB_OUTPUT_URL_TEMPLATE', '/adminjoboutput?job_id=<jobId>');
 oppia.constant(
   'ADMIN_TOPICS_CSV_DOWNLOAD_HANDLER_URL', '/admintopicscsvdownloadhandler');
+oppia.constant('ADMIN_SHOW_UPDATE_ROLE', false);
```

Comment:
This constant is added for showing the update role card. Change it to true to see the update role card.